multicast/__init__.py

```
@@ -498,21 +498,21 @@
 			)
 			calling_parser_group.add_argument(
-				"""--use-std""", dest='is_std', default=False, action='store_true'
+				"--use-std", dest="is_std", default=False, action="store_true"
 			)
 			calling_parser_group.add_argument(
-				"""--daemon""", dest='is_daemon', default=False, action='store_true'
+				"--daemon", dest="is_daemon", default=False, action="store_true"
 			)
 		subparsers = calling_parser_group.add_subparsers(
-			title="Tools", dest="cmd_tool", help=str("""Sub-Commands."""), metavar="CMD"
+			title="Tools", dest="cmd_tool", help="Sub-Commands.", metavar="CMD"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)